### PR TITLE
Remove generated files

### PR DESCRIPTION
**Pull request by tomibennett**
### Proposed changes

* Removes generated files from control version
  * types definitions
  * front schema
* add cleaning routines to remove generated files from filesystem
* in backend, remove little duplication from start scripts

By removing these files from versioning, we gain **much improve merge and/or rebase experiences** without loosing any functionality or performance.

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

Although, it builds and run in local, pass the CI tests, **make sure it is ok for deployed builds**.

### CODE REVIEW

**Comment by tomibennett** [comment]
as for now, the build fails on the CI. 

I isolated the steps in `.drone.yml`.

```
[...]
      - yarn build:types
      - node builder/schema/schema.js
      - node build/script-generate-schema.js
[...]
```

and found out that the `node build/script-generate-schema.js` ends up with a segfault.

See below the end of the log

```
[...]
+ node builder/schema/schema.js

  build/script-generate-schema.js      10.5mb ⚠️
  build/nodecallspython-NGVBVB5U.node   1.1mb ⚠️

⚡ Done in 656ms
+ node build/script-generate-schema.js
Segmentation fault (core dumped)
```

Full log available [here](https://drone.opencti.io/OpenCTI-Platform/opencti/9509/1/11).

Further investigation ongoing.

**Comment by tomibennett** [comment]
> as for now, the build fails on the CI.
> 
> I isolated the steps in `.drone.yml`.
> 
> ```
> [...]
>       - yarn build:types
>       - node builder/schema/schema.js
>       - node build/script-generate-schema.js
> [...]
> ```
> 
> and found out that the `node build/script-generate-schema.js` ends up with a segfault.
> 
> See below the end of the log
> 
> ```
> [...]
> + node builder/schema/schema.js
> 
>   build/script-generate-schema.js      10.5mb ⚠️
>   build/nodecallspython-NGVBVB5U.node   1.1mb ⚠️
> 
> ⚡ Done in 656ms
> + node build/script-generate-schema.js
> Segmentation fault (core dumped)
> ```
> 
> Full log available [here](https://drone.opencti.io/OpenCTI-Platform/opencti/9509/1/11).
> 
> Further investigation ongoing.

EDIT: It's not an issue anymore

File: opencti-platform/Dockerfile [hunk 1]
```diff
@@ -20,11 +20,19 @@ WORKDIR /opt/opencti-build/opencti-graphql
 COPY opencti-graphql/package.json opencti-graphql/yarn.lock opencti-graphql/.yarnrc.yml ./
 COPY opencti-graphql/.yarn ./.yarn
 COPY opencti-graphql/patch ./patch
+COPY opencti-graphql/src/python/requirements.txt ./src/python/requirements.txt
```
Comment (RomuDeuxfois) [inline opencti-platform/Dockerfile:23]: For back-end build on circle-ci

File: opencti-platform/Dockerfile_circleci [hunk 2]
```diff
@@ -19,8 +19,14 @@ WORKDIR /opt/opencti-build/opencti-graphql
 COPY opencti-graphql/package.json opencti-graphql/yarn.lock opencti-graphql/.yarnrc.yml ./
 COPY opencti-graphql/.yarn ./.yarn
 COPY opencti-graphql/patch ./patch
+COPY opencti-graphql/src/python/requirements.txt ./src/python/requirements.txt
+
```
Comment (RomuDeuxfois) [inline opencti-platform/Dockerfile_circleci:23]: Use for build back-end on circle-ci

File: opencti-platform/opencti-graphql/.gitignore [hunk 3]
```diff
@@ -16,12 +24,18 @@ config/tanium.json
...
-test-results
+
+# logs
 logs
 yarn-error.log
+
+# generated
 graphql.schema.json
-__pycache__
+src/generated/graphql.ts
```
Comment (RomuDeuxfois) [inline opencti-platform/opencti-graphql/.gitignore:37]: Added

**Comment by SamuelHassine** [comment]
@Kedae What we do with this one?

**Comment by SamuelHassine** [comment]
9 months ahead, closing....

**Pull request closed without merging**